### PR TITLE
REL-3966 Draw HRIFU Sky position

### PR DESCRIPTION
**Pull request by cquiroz**
image

### CODE REVIEW

File: bundle/jsky.app.ot/src/main/scala/jsky/app/ot/tpe/feat/TpeGhostIFUFeature.scala [hunk 1]
```diff
@@ -137,10 +137,9 @@ final class TpeGhostIfuFeature extends TpeImageFeature("GHOST", "Show the patrol
 
               // IFU2
               val p2 = pm.getLocationFromTag(s)
-              // We don't draw the SRIFU2, apparently, even though its position
-              // determines the position of the HR sky fibers
-              // drawStandardResolutionIfu(g2d, p2, Ifu2, AsSky)
-              drawHighResolutionSky(g2d, p2)
+              // We draw the SRIFU2 as Sky
+              drawStandardResolutionIfu(g2d, p2, Ifu2, AsSky)
+              //drawHighResolutionSky(g2d, p2)
```
Comment (cquiroz) [inline bundle/jsky.app.ot/src/main/scala/jsky/app/ot/tpe/feat/TpeGhostIFUFeature.scala:142]: I was a bit unsure whether to leave this on but it seems odd to draw two markers at the same position

**Review by sraaphorst — APPROVED** [review]
Looks fine to me. I guess this means we need a new test release of the OT built?

File: bundle/jsky.app.ot/src/main/scala/jsky/app/ot/tpe/feat/TpeGhostIFUFeature.scala (hunk 1 above)
Comment (swalker2m) [reply bundle/jsky.app.ot/src/main/scala/jsky/app/ot/tpe/feat/TpeGhostIFUFeature.scala:142]: Do the appear at the same place?  I thought the HR sky fibers were slightly offset.  At any rate, we can ask for feedback with this version.

Comment (cquiroz) [reply bundle/jsky.app.ot/src/main/scala/jsky/app/ot/tpe/feat/TpeGhostIFUFeature.scala:142]: Yes they show in the same place:
image

**Pull request merged**